### PR TITLE
chore: fix e2e cleanup by introduce new helper function ResetEnvdApp

### PR DESCRIPTION
**Pull request by aseaday**
https://github.com/tensorchord/envd/issues/802
Signed-off-by: nullday <aseaday@hotmail.com>

### CODE REVIEW

**Comment by aseaday** [comment]
I think it was caused by quick-start test, But `ir.DefaultGraph=new Graph` seems a little wired and ugly. Could we have a better way?

**Comment by kemingy** [comment]
I tried to add it to https://github.com/tensorchord/envd/blob/5c4971b2f5fa6b32c2c247e18cf6c6a178d28f57/pkg/lang/ir/compile.go#L88-L90 but sometimes it doesn't work as expected.

**Comment by kemingy** [comment]
BTW, you can search "pip install" in the e2e test log. You will find "pip install numpy numpy numpy".

**Comment by aseaday** [comment]
> BTW, you can search "pip install" in the e2e test log. You will find "pip install numpy numpy numpy".

Yeah, That what I want to find out a clean way rather than dirty work one by one to fix it. Let me check the interpret logic because I refact it last week

File: e2e/quick_start_test.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ package e2e
 import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
```
Comment (gaocegege) [inline e2e/quick_start_test.go:19]: Please add a blank line below.

Comment (aseaday) [reply e2e/quick_start_test.go:19]: Thx and got it. Wait me to work out a clean way than I fix it.

Comment (gaocegege) [reply e2e/quick_start_test.go:19]: Sure, take your time.

**Review by gaocegege — APPROVED** [review]
/lgtm
/approve

**Review by gaocegege — APPROVED** [review]

**Pull request merged**